lib/session/session.go

```diff
@@ -384,21 +385,21 @@ func (s *server) UpdateSession(req UpdateRequest) error {
 // NewTerminalParamsFromUint32 returns new terminal parameters from uint32 width and height
 func NewTerminalParamsFromUint32(w uint32, h uint32) (*TerminalParams, error) {
```

Comment:
I don't think it's a job of this funciton to make a decision to set to default. I would keep it as is and will let caller to decide to resort to defaults, or create a separate functions with explicit semantics